cosmo_tester/framework/cfy_helper.py

```diff
@@ -97,6 +101,34 @@ def bootstrap(self,
                 task_retry_interval=task_retry_interval,
                 verbose=verbose,
                 debug=debug).wait()
+            self.upload_plugins(cfy)
```

Comment:
It has to be here, as the simple_manager_bootstrap tests call this (because they bootstrap using the simple manager blueprint on another machine).